pkg/oc/cli/cmd/projects.go

```diff
@@ -113,19 +112,20 @@ func (o *ProjectsOptions) Complete(f kcmdutil.Factory, args []string, commandNam
 	}
 	o.Client = projectClient.Project()
 
-	o.Out = out
+	return nil
+}
 
+func (o *ProjectsOptions) Validate(args []string) error {
+	if len(args) > 0 {
+		return fmt.Errorf("no arguments should be passed")
+	}
 	return nil
 }
 
 // RunProjects lists all projects a user belongs to
 func (o ProjectsOptions) RunProjects() error {
-	config := o.Config
-	clientCfg := o.ClientConfig
-	out := o.Out
-
 	var currentProject string
-	currentContext := config.Contexts[config.CurrentContext]
+	currentContext := o.Config.Contexts[o.Config.CurrentContext]
```

Comment:
You changed this from a shallow copy to the same object.  Are you sure that's safe/correct?

Reply:
Thanks, will revert this change